manim/config/main_utils.py

```diff
@@ -422,4 +422,12 @@ def _parse_args_no_subcmd(args):
         metavar="True/False",
     )
 
+    # Specify the directory where assets are located
+    parser.add_argument(
+        "--assets_dir",
+        type=str,
+        help="Use this folder when using relative paths rather "
+        "than the current working directory where manim was invoked",
```

Comment:
This sounds incorrect to me. Not all paths will be considered relative to assets_dir. The only paths that will be relative to it are the ones where manim searches for external resources.

Reply:
I agree, my definition is not descriptive enough. How's:
```
        help="Use this folder path to locate resource files "
        "(.svg/.png/etc...) in addition to the current working directory "
        "where manim was invoked"
```

Reply:
I'd rather keep the help as short as possible: "External resources (e.g. svg/png files) are to be found in the working directory or in this path."

Sorry to be fussy about this one, but help strings are one of the first things that the user may see, so we want them to be really crisp :)